tests/subgraph/minter-suite/minter-suite-mapping.test.ts

```diff
@@ -3114,16 +3064,16 @@ describe("MinterHolder-specific tests", () => {
       event.block.timestamp = CURRENT_BLOCK_TIMESTAMP;
 
       if (minterType == "MinterHolderV0") {
-        handleUnregisteredNFTAddressV0(
-          changetype<HolderV0UnregisteredNFTAddress>(event)
+        handleUnregisteredNFTAddress(
+          changetype<HolderUnregisteredNFTAddress>(event)
         );
       } else if (minterType == "MinterHolderV1") {
-        handleUnregisteredNFTAddressV1(
-          changetype<HolderV1UnregisteredNFTAddress>(event)
+        handleUnregisteredNFTAddress(
+          changetype<HolderUnregisteredNFTAddress>(event)
         );
       } else {
-        handleUnregisteredNFTAddressV2(
-          changetype<HolderV2UnregisteredNFTAddress>(event)
+        handleUnregisteredNFTAddress(
+          changetype<HolderUnregisteredNFTAddress>(event)
         );
       }
```

Comment:
Can be follow-on work, but I think we can get rid of these if statements altogether, right?

Reply:
resolving this because I don't want to hold up merging, and will take a look at this for follow-on activity!

Reply:
Yeah we can def get rid of these, I just got lazy with the test updates